tests/game.js

```diff
@@ -13,65 +14,77 @@ const cards = [
     { key: 'wolf_cub', value: -8, amount: 1 }
 ];
 
+const language = 'en';
+
 describe('Game', () => {
     const players = 25;
 
+
     it('should have both modes.', () => {
         game.mode.should.have.property('NORMAL');
         game.mode.should.have.property('CHAOS');
     });
 
-    let normalGame = {};
-    const language = 'en';
-    it('should create a normal game.', () => {
-        normalGame = game.create(players, language, cards, game.mode.NORMAL);
-        normalGame.should.not.be.empty();
-    });
+    describe('Create', () => {
+        describe('given mode is NORMAL', () => {
+            beforeEach(function () {
+                this.game = game.create(players, language, cards, game.mode.NORMAL);
+            });
 
-    it('should contain the weight on each normal game created.', () => {
-        normalGame.should.have.property('weight').which.is.a.Number();
-    });
+            it('should create a normal game.', function () {
```

Comment:
I know there was a limitation of using arrow functions in mocha's tests but I can't remember it. This was the reason behind your refactor changing the functions?